client/web/compose/src/components/faIcons.js

```diff
@@ -62,6 +62,7 @@ import {
   faSync,
   faExclamationTriangle,
   faEllipsisV,
+  faLocationArrow,
```

Comment:
Since we dont have the location icon in free version, use the compass icon(fa-compass)